package.json

```diff
@@ -9,118 +9,85 @@
     "Blockchain",
     "Security"
   ],
-  "main": "dist/mythxjs.umd.js",
-  "module": "dist/mythxjs.es5.js",
-  "typings": "dist/types/mythxjs.d.ts",
-  "files": [
-    "dist"
-  ],
-  "author": "Eswara Sai <eswarasai.v@gmail.com>",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/ConsenSys/mythxjs.git"
-  },
+  "author": "Eswara Sai @eswarasai",
+  "homepage": "https://github.com/ConsenSys/mythxjs",
+  "repository": "https://github.com/ConsenSys/mythxjs.git",
+  "bugs": "https://github.com/ConsenSys/mythxjs/issues",
   "license": "MIT",
+  "dependencies": {
+    "@oclif/command": "^1",
+    "@oclif/config": "^1",
+    "@oclif/plugin-help": "^2",
+    "axios": "^0.18.0",
+    "cli-table": "^0.3.1",
+    "cli-ux": "^5.2.1",
+    "jsonwebtoken": "^8.5.1",
+    "tslib": "^1"
+  },
+  "devDependencies": {
+    "@oclif/dev-cli": "^1",
+    "@oclif/test": "^1",
+    "@oclif/tslint": "^3",
+    "@types/chai": "^4",
+    "@types/cli-table": "^0.3.0",
+    "@types/jsonwebtoken": "^8.3.2",
+    "@types/mocha": "^5",
+    "@types/node": "^10",
+    "chai": "^4",
+    "globby": "^8",
```

Comment:
Where is this package being used?

Reply:
They are dev dependencies for the oclif framework